### PR TITLE
[python] Outgest `uns`

### PR DESCRIPTION
**Pull request by johnkerl**
**Issue and/or context:** #1764

**Changes:**

**Notes for Reviewer:** This PR is ready for review.

### CODE REVIEW

**Comment by nguyenv** [comment]
I did see this PR a few days ago but noticed some `TODO comments` so didn't review yet. Are is that meant to be done in this PR or in a subsequent one?

File: apis/python/src/tiledbsoma/io/ingest.py [hunk 1]
```diff
@@ -2120,15 +2130,17 @@ def _ingest_uns_node(
         return
 
     if isinstance(value, pd.DataFrame):
-        num_cols = value.shape[1]
+        ####num_cols = value.shape[1]
```
Comment (nguyenv) [inline apis/python/src/tiledbsoma/io/ingest.py:2133]: Do these need to be removed?

File: apis/python/src/tiledbsoma/io/ingest.py [hunk 2]
```diff
@@ -2568,3 +2604,81 @@ def _extract_obsm_or_varm(
...
+                    + "{_UNS_OUTGEST_HINT_KEY} as unrecognized {hint}: leaving this as Pandas DataFrame"
+                )
+                logging.log_io_same(msg)
+                extracted[key] = pdf
+        elif isinstance(element, SparseNDArray):
+            extracted[key] = element.read().tables().concat().to_pandas()
+        elif isinstance(element, DenseNDArray):
+            extracted[key] = element.read().to_numpy()
+        else:
+            print("SKIPPING", element.soma_type)
```
Comment (nguyenv) [inline apis/python/src/tiledbsoma/io/ingest.py:2641]: Use `print` or `logging` here?

**Review by nguyenv — APPROVED** [review]

**Comment by johnkerl** [comment]
Thanks @nguyenv ! :)

**Pull request merged**